### DISCUSSION TITLE
Package deprecation request(s)

### DISCUSSION THREAD

**Discussion by rwojsznis** [community/community#129783] [npm]
Hey folks!

There are some packages that are clearly deprecated, such as [flux](https://www.npmjs.com/package/flux), but they are not officially marked as [deprecated via npm itself](https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions). As a result, tools that manage dependencies (like Renovate) can't detect or report these libraries and inform users that they should look for replacements.

The problem is that GitHub repositories are often in read-only mode (like the flux one), and the original authors have moved on to different projects or companies. This makes it difficult to ask them directly to properly deprecate their past work.

Is there a way that npmjs.com could deprecate such projects for the sake of clarity? Maybe you could offer extra drop-down entry for that [on your support page](https://www.npmjs.com/support)? 🤔

**Comment by RachidZahrani** [comment]
Some packages, like flux, are deprecated but not officially marked, making it hard for tools like Renovate to detect them.
Allow npmjs.com to let the community or npm team mark packages as deprecated for clarity.

Implementation:
Add a feature on npmjs.com for submitting deprecation requests.
Include an option on the npm support page to request package deprecation.
